main.js

```diff
@@ -0,0 +1,111 @@
+const express = require('express')
+// Get the client
+const bodyParser = require('body-parser');
```

Comment:
it would be good to see how we can also collect the data for each anomaly detected and store the relevant info for it in the db so that we can analyse it later on

Reply:
Not done yet. Will see if I get time to do this as well.